runtime/queries/metricsview_resolve_time_ranges.go

```diff
@@ -0,0 +1,108 @@
+package queries
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"github.com/rilldata/rill/runtime"
+	"github.com/rilldata/rill/runtime/pkg/rilltime"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type MetricsViewTimeRanges struct {
```

Comment:
This basically just a) calls the timestamps resolver (which may be cached in the resolver cache), b) calls `rilltime`, which doesn't involve heavy operations and should be fast. So can the logic here just be moved directly to the API handler for `MetricsViewTimeRanges`? Since there is no need for separate caching of the result here.

It's also nice not to add new logic to the `runtime/queries` package since it's being deprecated.